log4j-layout-template-json/src/test/java/org/apache/logging/log4j/layout/template/json/JsonTemplateLayoutNullEventDelimiterTest.java

```diff
@@ -71,13 +71,24 @@ public void test() throws Exception {
             // Verify the received log events.
             final byte[] actualBytes = server.getReceivedBytes();
             Assertions.assertThat(actualBytes).startsWith(expectedBytes);
-
-        } catch (BindException bindException) {
-            System.err.println("Required port " + PORT + " not available, skipping " + getClass().getSimpleName());
+        } finally {
+            tempConfig.delete();
         }
 
     }
 
+    public void makeTempConfig(final int serverPort, final File tempConfigFile) throws IOException {
```

Comment:
This approach looks way better, thanks @perry2of5! Would you use `final` modifier on variables, wherever possible, please?

Reply:
Yes.